themes/bootstrap3/copyDependencies.mjs

```diff
@@ -0,0 +1,19 @@
+import { copyFile } from 'node:fs/promises';
```

Comment:
Should we create a tools or util directory to hold this script instead of putting it in the root of the theme? I think it might be worth establishing some kind of pattern so we don't end up with lots of files accumulating in the theme root over time, if we add other things like this in the future.

Reply:
Done!